airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py

```diff
@@ -156,12 +156,12 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
             query = single_query_config["query"]
             if self.is_metrics_in_custom_query(query):
                 if non_manager_accounts:
-                    if query.resource_name in FULL_REFRESH_CUSTOM_TABLE:
+                    if query.resource_name in FULL_REFRESH_CUSTOM_TABLE or single_query_config.get("full_refresh_only"):
```

Comment:
Need to add unit tests, bump version and update change log.

Reply:
@lazebnyi can you help me understand what additional tests would be valuable? we have a unit test for checking that `is_metrics_in_custom_query` returns a valid response [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-google-ads/unit_tests/test_source.py#L153-L165)